src/app/page.tsx

```diff
@@ -8,7 +10,15 @@ export default function HomePage() {
   const heroImageUrl = getAssetUrl(homePage?.fields.homepageHeroImage);
 
   return (
-    <div className="mx-auto max-w-7xl space-y-16 py-8 px-4">
+    <div className="mx-auto max-w-7xl space-y-6 py-8 px-4">
+      <div className="mx-auto max-w-5xl space-y-1 text-lg text-center">
+        <p>
+          Abhivyakti is Los Angeles&apos; premier Marathi theater group,
+          dedicated to bringing powerful performances and storytelling to
+          the stage. Our mission is to preserve and promote Marathi
+          culture and theatrical arts in Southern California.
+        </p>
+        </div>
```

Comment:
We should add a CI check for Prettier. You should change your IDE settings to "format-on-save".